### PR TITLE
Boot `cider-jack-in-clojurescript` instructions

### PR DESCRIPTION
**Pull request by pbzdyl**

### CODE REVIEW

**Comment by bbatsov** [comment]
Did you check if this is properly rendered locally? I see you've used different indentation levels for the code blocks.

**Comment by pbzdyl** [comment]
Yes, I was editing it directly on GitHub. You can preview it at https://github.com/pbzdyl/cider/blob/patch-1/doc/up_and_running.md.

**Comment by bbatsov** [comment]
The problem is that mkdocs is not using the same markdown parser. Some formatting that work on GitHub don't work with mkdocs, so it's always best to try mkdocs directly. (http://cider.readthedocs.org/en/latest/about/contributing/#working-on-the-manual).

**Comment by Malabarba** [comment]
After Jack in, what did you run to start the cljs repl? Was it one of the snippets in cider-cljs-lein-repl, or was it something else?

**Comment by pbzdyl** [comment]
@Malabarba I used `(start-repl)`.

**Comment by pbzdyl** [comment]
@bbatsov I have fixed the indentation. However, I cannot get the numbering work correctly but it seems it is a global issue in the docs (if a numbered paragraph contains code snippets the numbering starts from 1 in the following numbered paragraph).

**Comment by bbatsov** [comment]
Guess something should be formatted different with mkdocs, I'm not sure what. Just replace the numbers with something else - e.g. no numbers at all.

File: doc/up_and_running.md [hunk 1]
```diff
@@ -77,19 +77,29 @@ ClojureScript support relies on the
...
 
 ```clojure
 :repl-options {:nrepl-middleware [cemerick.piggieback/wrap-cljs-repl]}
 ```
 
+   In `built.boot`:
+
+```clojure
+(task-options!
+  repl {:middleware '[cemerick.piggieback/wrap-cljs-repl]})
```
Comment (Malabarba) [inline doc/up_and_running.md]: shouldn't this be :repl?

**Comment by Malabarba** [comment]
> @Malabarba I used (start-repl).

Do you have any extra code that gets run before that?
I tried to follow the instructions, then call `start-repl` in a repl, but I get a nasty null pointer exception. :-(

**Comment by pbzdyl** [comment]
@Malabarba I will create a sample project that works on machine and post it online tomorrow.

**Comment by pbzdyl** [comment]
@bbatsov I have changed the formatting so there is no list. It seems that mkdocs doesn't support code blocks in lists and it breaks the formatting: https://github.com/waylan/Python-Markdown/issues/53.

**Comment by bbatsov** [comment]
Hopefully we won't use mkdocs very long. @annapawlicka volunteered recently to port to manual to sphinx.

**Comment by annapawlicka** [comment]
Expect incremental updates starting this week 🔨

**Comment by pbzdyl** [comment]
@Malabarba I have posted a sample project: https://github.com/pbzdyl/cider-boot-cljs.

I tried `cider-jack-in-clojurescript` with it and it worked - I got cljs repl working. I guess it is a matter of dependencies required by `boot-cljs-repl`: https://github.com/adzerk-oss/boot-cljs-repl#boot-cljs-repl

**Pull request merged**